src/egon/data/datasets/hydrogen_etrago/bus.py

```diff
@@ -35,26 +35,33 @@ def insert_hydrogen_buses(scenario="eGon2035"):
 
     Parameters
     ----------
-    scenario : str, optional
-        Name of the scenario, the default is 'eGon2035'.
+    No parameter is required.
 
     """
-    sources = config.datasets()["etrago_hydrogen"]["sources"]
-    target = config.datasets()["etrago_hydrogen"]["targets"]["hydrogen_buses"]
-    # initalize dataframe for hydrogen buses
-    carrier = "H2_saltcavern"
-    hydrogen_buses = initialise_bus_insertion(
-        carrier, target, scenario=scenario
-    )
-    insert_H2_buses_from_saltcavern(
-        hydrogen_buses, carrier, sources, target, scenario
-    )
-
-    carrier = "H2_grid"
-    hydrogen_buses = initialise_bus_insertion(
-        carrier, target, scenario=scenario
-    )
-    insert_H2_buses_from_CH4_grid(hydrogen_buses, carrier, target, scenario)
+    s = config.settings()["egon-data"]["--scenarios"]
+    scn = []
+    if "eGon2035" in s:
+        scn.append("eGon2035")
+    if "eGon100RE" in s:
+        scn.append("eGon100RE")
```

Comment:
I can somehow not find where this variable "scn" is used later on.

Reply:
Fixed in 9b254724f35f014ec7074057448ad2ce5ff7d3d6